src/Microsoft.Framework.Caching.SqlServer/project.json

```diff
@@ -0,0 +1,28 @@
+{
+    "commands": {
+        "sqlservercache": "Microsoft.Framework.Caching.SqlServer"
+    },
+    "dependencies": {
+        "Microsoft.Framework.Caching.Abstractions": "1.0.0-*",
+        "Microsoft.Framework.CommandLineUtils.Sources": { "version": "1.0.0-*", "type": "build" },
+        "Microsoft.Framework.Logging": "1.0.0-*",
+        "Microsoft.Framework.Logging.Console": "1.0.0-*",
+        "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" },
+        "Microsoft.Framework.OptionsModel": "1.0.0-*"
+    },
+    "description": "ASP.NET 5 distributed cache implementation on Microsoft SQL Server.",
+    "frameworks": {
```

Comment:
Out of curiosity, have you considered adding a `net45(1)` target like `Microsoft.Framework.Caching.Abstractions`?

https://github.com/aspnet/Caching/blob/dev/src/Microsoft.Framework.Caching.Abstractions/project.json